app/dashboard/views.py

```diff
@@ -5538,6 +5572,11 @@ def fulfill_bounty_v1(request):
     if bounty.event:
         try:
             project = HackathonProject.objects.get(pk=request.POST.get('projectId'))
+            demo_provider = request.POST.get('videoDemoProvider')
+            demo_link = request.POST.get('videoDemoLink')
+            if demo_link:
+                project.extra['video_url'] = demo_link
+                project.extra['video_provider'] = demo_provider if demo_provider in ('loom', 'youtube', 'vimeo') else 'generic'
```

Comment:
do we need the user to pick the provider? why not just do a regex to avoid users input errors and less fields on the form?

Reply:
Do you think is better to remove the provider select button? I think another option is to deactivate the button and display the provider based on the regex. Also in the current implementation regexs are used to determine the provider but the user can change it if he/she thinks the provider was wrong picked.

Reply:
yes based on regex should be better, the user shouldn't need to decide as the info is already on the link, if the link is not supported then let the user know. Sounds good?

Reply:
Are we going to make this change soon so we can get this merged in? @zoek1

Reply:
done @danlipert @octavioamu